content/applications/productivity/knowledge/properties.rst

```diff
@@ -2,101 +2,102 @@
-       creation.
-
-       To set it up, click :guilabel:`Add a Value` next to the :guilabel:`Values` field. Enter
-       predetermined values and press **enter** to validate; you can enter as many values as needed.
-       Click anywhere to close the property creation window.
+     - To add a drop-down selection menu with choices, click :guilabel:`Add a Value`, type the name
+       of the choice, and press `Enter` to save it. Repeat this for each value.
    * - :guilabel:`Tags`
-     - Allows creating and applying as many tags as needed.
-
-       To set it up, enter your `new_tag` in the :guilabel:`Tags` field, and press **enter** or
-       click :guilabel:`Create "new_tag"`. Click anywhere to close the window. Then, add the tags
-       into the property field. To do so, click the property field and choose from the created tags;
-       enter the tags' name and press **enter**; enter a new tag's name and create a new one on the
-       spot.
+     - To create tags, type a tag name in the :guilabel:`Tag` field and press `Enter` or click
+       :guilabel:`Create "new_tag"` to save it. Click on a tag to select a different color for it.
    * - :guilabel:`Many2one`
-     - Choose from a list of records that result from a model's domain. You can only select one
-       result.
-
-       To set it up, click :guilabel:`Search a Model` in the :guilabel:`Model` field, select the
-       model. Match all records by clicking :guilabel:`## Record(s)`, or filter the results by
-       clicking :guilabel:`+ Add Filter` and show the records by clicking :guilabel:`## Record(s)`.
+     - Select one 0 or 1 record from a different model as the field value. Configure by clicking
+       inside the :guilabel:`Model` field, then select the model type from which to choose a record.
+       After saving the property, click inside the property field to manually select values, which
+       replaces the previous selection.
    * - :guilabel:`Many2many`
-     - Choose from a list of records that result from a model's domain. You can select as many
-       results as needed.
+     - Select multiple records from a different model as the field value. Configure by clicking the
+       :guilabel:`Model` field, then select the model type from which to choose any number of
+       records. After saving the property, click inside the property field to manually select
+       various values manually.
 
-       To set it up, click :guilabel:`Search a Model` in the :guilabel:`Model` field, select the
-       model. Match all records by clicking :guilabel:`## Record(s)`, or filter the results by
-       clicking :guilabel:`+ Add Filter` and show the records by clicking :guilabel:`## Record(s)`.
+.. note::
+   - Click outside the property field window to save a property.
+   - With the :guilabel:`Many2one` and :guilabel:`Many2many` field types, editing the
+     :guilabel:`Domain` restricts the choice of values, while adding a :guilabel:`Default value`
```

Comment:
seealso to: https://www.odoo.com/documentation/18.0/applications/essentials/search.html#custom-filters